pkg/agent/node/node.go

```diff
@@ -32,17 +32,15 @@ func Params() map[string]interface{} {
 		"requestedHostname": os.Getenv("CATTLE_NODE_NAME"),
 	}
 
-	for k, v := range params {
-		if m, ok := v.(map[string]string); ok {
-			for k, v := range m {
-				logrus.Infof("Option %s=%s", k, v)
-			}
-		} else {
-			logrus.Infof("Option %s=%v", k, v)
+	dclient, err := client.NewClientWithOpts(client.WithAPIVersionNegotiation())
+	if err != nil {
+		logrus.Errorf("Error getting client with API version negotiation, using default client from Env: %v", err)
```

Comment:
Can we use Docker client here to make sure everyone gets what is meant by client?